compiler/rustc_codegen_ssa/src/back/symbol_export.rs

```diff
@@ -93,16 +93,11 @@ fn reachable_non_generics_provider(tcx: TyCtxt<'_>, _: LocalCrate) -> DefIdMap<S
                 return None;
             }
 
-            // Functions marked with #[inline] are codegened with "internal"
```

Comment:
Why'd you remove the comment? Seems valid still to mention the interaction w/ inline.

Reply:
I removed it because the logic for the `inline` attribute and extern indicators is in `cross_crate_inlinable`. I can put back a comment if you want :shrug: but one of the purposes of `tcx.cross_crate_inlinable` is to be the single point in the compiler that juggles those interactions.